roles/user-management/manage-atlassian-users/README.md

```diff
@@ -11,28 +11,34 @@ This is a sample of a YAML file containing the variables.
 
 ```
 ---
-atlassian_url: https://test.atlassian.net
-atlassian_username: admin
-atlassian_password: admin
-atlassian_users:
-  - firstname: Test
-    lastname: 123
-    password: password123
-    email: test@example.com
-    groups:
+atlassian:
+  url: https://example.atlassian.net
+  username: example
+  password: example
+
+  create:
```

Comment:
I don't think the `create` is needed here. We should rather adopt the typical `ansible` way of using `state` - i.e.: `state: present` v.s. `state: absent`, etc.